examples/tests/Earthfile

```diff
@@ -200,3 +203,23 @@ dockerfile-test:
         --entrypoint \
         --mount=type=tmpfs,target=/tmp/earthly \
         -- --no-output +test
+
+fail-test:
+    COPY fail.earth ./Earthfile
+    # test that an error code is correctly returned
+    RUN --privileged ! /usr/bin/earth-buildkitd-wrapper.sh +test
```

Comment:
Oh - another detail. I've noticed it uses less cache if the buildkit dir is not preserved (`--mount=type=tmpfs,target=/tmp/earthly`). See other tests.

Reply:
I was wondering why those other tests had that, that makes a lot of sense.